cmd/backend/build_training_env/build-training-env.go

```diff
@@ -0,0 +1,78 @@
+package build_training_env
```

Comment:
non-blocking: might wanna rename this. `build-training-env` at face value could either mean `.env` file or `environment`. In this case, this is about `.env` file.

Reply:
good point.